tests/ci_commands_script.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# datadict and datadictwithobjid tests must run separately to allow
+# loading different datamodels
+poetry run pytest -vv --cov=sheepdog --cov-report xml tests/integration/datadict
+
+# since this whole thing is run as a bash {{this script}}, only the last pytest
+# command controls the exit code. We actually want to exit early if something fails,
+# so check that here and exit as necessary
+RESULT=$?
+if [ $RESULT -ne 0 ]; then
+  exit 1
```

Comment:
would it work to just `set -e` at the beginning of the script?

Reply:
Done and I tested it works by purposing failing a test in the middle run :+1: